docs/sources/explore/get-started-with-explore.md

```diff
@@ -0,0 +1,210 @@
+---
+aliases:
+  - features/explore/
+keywords:
+  - explore
+  - loki
+  - logs
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Get started with Explore
+weight: 5
+---
+
+# Get started with Explore
+
+Explore is your gateway for querying, analyzing, and aggregating data in Grafana. It allows you to visually explore and iterate until you develop a working query or set of queries for building visualizations and conducting data analysis. If your data source supports graph and table data, there's no need to create a dashboard, as Explore can display the results in both formats. This facilitates quick, detailed, real-time data analysis.
+
+With Explore you can:
+
+- Create visualizations to integrate into your dashboards.
+- Create queries using mixed data sources.
+- Create multiple queries within a single interface.
+- Understand the shape of your data across various data sources.
+- Perform real time data exploration and analysis.
+
+Key features include:
+
+- Query editor, based on specific data source, to create and iterate queries.
+- [Query history](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/query-management/) to track and maintain your queries.
+- [Query inspector](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/explore-inspector/) to help troubleshoot query performance.
+
+Watch the following video to get started using Explore:
+
+{{< youtube id="1q3YzX2DDM4" >}}
+
+## Requirements
```

Comment:
Task topics use "Before you begin": https://grafana.com/docs/writers-toolkit/structure/topic-types/task/

```suggestion
## Before you begin
```